src/stars__stars.css

```diff
@@ -0,0 +1,33 @@
+.stars__star {
+  content: " ";
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 12px;
+  height: 12px;
```

Comment:
```suggestion
  width: 16px;
  height: 16px;
```
![image](https://github.com/mate-academy/layout_stars/assets/80700247/ebb34af9-d386-450d-837d-60ae833628cf)

Reply:
Figma file is different than reference file in test. You can't pass npm tests when you use Figma as reference